### DISCUSSION TITLE
How to use alias like @ or ~ for importing or auto import?

### DISCUSSION THREAD

**Discussion by jdkcoder** [elysiajs/elysia#672] [Q&A] [answered]
```import db from '../../db'
import { tag } from '../../db/schemas'
```

How can I use alias like `@` or `~` instead of `../../`? 
Is there any ways to add auto import if I don't wanna to create every new `index.js` in each folder or if I don't wanna import every single file?

I'm using Bun v1.1.12 + Elysia v1.0.22

**Comment by cannap** [comment]
https://bun.sh/guides/runtime/tsconfig-paths

**Comment by jdkcoder** [comment, marked as answer]
![image](https://github.com/elysiajs/elysia/assets/89140442/9bf8535c-28d1-462c-ae11-0cc774a23f45)

I found a solution that fit me atm, accepted using index.js file
